src/client/styles/components/FilterPopup.scss

```diff
@@ -42,6 +42,7 @@
 
   svg {
     vertical-align: middle;
+    white-space: normal;
```

Comment:
This needs to be on the parent `<button>` not the `<svg>` 
sorry i wasnt clear about that... sad.

Reply:
Since you got it clear in the Design Toolkit, I won't be making this update.